package.json

```diff
@@ -27,6 +27,9 @@
     "lib",
     "umd"
   ],
+  "engines": {
```

Comment:
We this is a client side react-lib and `np` is a devDependency, let's remove this engines block. Also means we could get by calling this a patch update vs a major version

Reply:
np requires the engines block unfortunately. it's not enforced unless we add `engine-strict: true` as well.

how about we just drop it down to `node: >=14.0.0` for good measure?

Reply:
makes sense!